build.gradle

```diff
@@ -54,6 +54,13 @@ task yarnPackage(type: YarnTask, dependsOn: 'yarn') {
 }
 distTar.dependsOn(yarnPackage)
 
+task yarnLint(type: YarnTask, dependsOn: 'yarn') {
```

Comment:
is this task run somewhere? .travis.yam?

Reply:
Check now depends on yarnLint. So, Travis will run those indeed.